core/src/toga/platform.py

```diff
@@ -1,11 +1,14 @@
+from __future__ import annotations
+
 import importlib
 import os
 import sys
 from functools import lru_cache
+from types import ModuleType
 
-if sys.version_info >= (3, 10):
+if sys.version_info >= (3, 10):  # pragma: no cover
     from importlib.metadata import entry_points
-else:
+else:  # pragma: no cover
```

Comment:
This doesn't seem right - we shouldn't need to `# prama` these - and we *definitely* don't need to no cover *both* of them.

Reply:
Ah - this is because we don't have the conditional coverage handling from Briefcase.